androidApp/src/main/java/com/mbta/tid/mbta_app/android/routeDetails/RouteStopListView.kt

```diff
@@ -364,8 +364,10 @@ private fun RouteStops(
         horizontalAlignment = Alignment.Start,
         scrollEnabled = !loading,
     ) {
+        val hasTypicalSegment = stopList.segments.any { it.isTypical }
```

Comment:
As [discussed in slack](https://mbta.slack.com/archives/C062QNAJZ2M/p1752863038564189?thread_ts=1752848976.319869&cid=C062QNAJZ2M), this is a simplified approach to solving this problem which requires the assumption that when there is no pattern of typicality 1, then all patterns are of the same typicality.